ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/cv-pathfinder-edge-wan-use-evpn-on-lan.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Testing CV pathfinder edge using new toggle wan_use_evpn_node_settings_for_lan
+wan_use_evpn_node_settings_for_lan: true
```

Comment:
Currently the config is not good, since evpn on both sides leads to wan vrf being send to the lan even when the address_families are []. We know this and it will change with the gateway feature.

Reply:
correct